pkg/target/locker.go

```diff
@@ -33,15 +33,15 @@ type Locker interface {
 	// leaves the existing locks untouched in case of conflicts.
 	// Locks are reentrant, locking existing locks (with the same owner)
 	// extends the deadline.
-	Lock(types.JobID, []*Target) error
+	Lock(jobID types.JobID, jobTargetManagerAcquireTimeout time.Duration, targets []*Target) error
```

Comment:
Can you rename this here, the Lock API shouldn't know/care if this is used by a target manager or something else. Just name it livetime or duration or something like that.

Also please fix the sentence 3 lines above that says the timeout is set at construction time :)

(same below for `TryLock`)

Reply:
Aliegned comments.